spec/factories/rents.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :rent do
+    user { '' }
+    book { '' }
```

Comment:
mmm... please add factory [associations](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations).

Reply:
did it